flytekit/deck/deck.py

```diff
@@ -79,8 +93,8 @@ class TimeLineDeck(Deck):
     Instead, the complete data set is used to create a comprehensive visualization of the execution time of each part of the task.
     """
 
-    def __init__(self, name: str, html: Optional[str] = ""):
-        super().__init__(name, html)
+    def __init__(self, name: str, html: Optional[str] = "", auto_add_to_deck: bool = False):
```

Comment:
I think it can be confusing to have a different default here. Can this be the same default as `Deck`, but have the caller set `auto_add_to_deck=False`?

```suggestion
    def __init__(self, name: str, html: Optional[str] = "", auto_add_to_deck: bool = True):
```

Reply:
Done, thanks for the advice!